sharepoint/sharepoint-ps/sharepoint-online/New-SPOSiteFileVersionBatchDeleteJob.md

```diff
@@ -16,7 +16,7 @@ manager: seanmc
 ## SYNOPSIS
 
 > [!NOTE]
-> This feature is part of the Version history controls Preview. If your tenant is not part of the Preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
 
 Starts a file version batch trim job targeting all document libraries in a site collection.
```

Comment:
Queues a job to trim versions for a document library in a site collection. 

Caution: Versions deleted using this cmdlet will be permanently deleted and cannot be recovered from the recycle bin.

Reply:
Added "Queues a job to trim versions for all document libraries in a site collection.", added all in the description.